static_src/main.js

```diff
@@ -32,7 +32,10 @@ function dashboard() {
 function org(orgGuid) {
   orgActions.changeCurrentOrg(orgGuid);
   cfApi.fetchOrg(orgGuid);
-  React.render(<App><SpaceList initialOrgGuid={ orgGuid } /></App>, mainEl);
+  React.render(
+    <App currentOrgGuid={ orgGuid }>
```

Comment:
Why do we need to store the org guid in the app tag? and why not also the space or app guid